src/main/java/com/puppycrawl/tools/checkstyle/checks/annotation/AnnotationLocationCheck.java

```diff
@@ -130,14 +137,12 @@
  * &lt;/module&gt;
  * </pre>
  * <p>
- * Example to allow only one and only parameterized annotation on the same line
+ * Example to allow one single parameterless annotation on the same line is same as
+ * the example provided for default configuration
```

Comment:
Please add a period `.` at the end since there is no example after this statement.

Reply:
Done